
error.generic.exception=Houve um problema com nossos servi\u00e7os, tente novamente mais tarde.
error.cpf.already.used=CPF {0} j\u00e1 cadastrado
error.elector.not.found=Eleitor com id {0} n\u00E3o encontrado.

